src/components/ProductCard/index.js

```diff
@@ -0,0 +1,34 @@
+import Image from 'next/image';
+
+import useStore from '../../hooks/useStore';
+import StyledButton from '../Button/styled';
+import StyledProductName from '../ProductName/styled';
+import StyledProductPrice from '../ProductPrice/styled';
+
+import StyledProductCard from './styled';
+
+export default function ProductCard({id, name, price, imgUrl, category}) {
+	const addToCart = useStore(state => state.addToCart);
+
+	return (
+		<StyledProductCard key={id} category={category}>
+			<Image
+				alt={'Photo from the product'}
```

Comment:
Same here about the alt text.
```suggestion
				alt=""
```